cmd/version/cmd.go

```diff
@@ -0,0 +1,14 @@
+package version
+
+import (
+	"github.com/spf13/cobra"
+	"numerous.com/cli/internal/version"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "version",
```

Comment:
Isn't it normally attached to the root command, so you can write `numerous -v` or `numerous --version`?

Reply:
Sometimes, yes. `git` for example allows both `git -v`, `git --version`, and `git version`. Other commands only have `version`, like `kubectl` for example. And of course most GNU tools have only `-v`.